sdk/keyvault/azure-keyvault-administration/README.md

```diff
@@ -1,15 +1,310 @@
-# Azure Key Vault Administration client library for Python
+# Azure KeyVault Administration client library for .NET
```

Comment:
Was Python supposed to be changed to .NET?

Reply:
oh jeez, thanks for catching all of my embarrassing copy-past errors

Reply:
that's what I'm here for ✨